### DISCUSSION TITLE
Video enderer

### DISCUSSION THREAD

**Discussion by dinushawwww** [AxisCommunications/discussions#99] [Axis Windows SDKs] [answered]
I have use AMC for video camera on C# application. the camera view is with noise and shows Video rendere 9 is used. How to improve the video ?
![image](https://user-images.githubusercontent.com/47741867/198504228-c8d92b06-477f-4b89-83b6-20720ed36205.png)

**Comment by anders-axis** [comment, marked as answer]
Hi,
For some reason the PC doesn't support Enhanced Video Renderer (default). Maybe this is a very old Windows version or server OS?

To avoid the error message just specify that Video Mixing Renderer 9 should be used:
`amc.VideoRenderer = (int)AMC_VIDEO_RENDERER.AMC_VIDEO_RENDERER_VMR9;`

The renderer will not impact image quality. Could you show an example of the noise?
